### DISCUSSION TITLE
Slack integration for private channels

### DISCUSSION THREAD

**Discussion by alexopenline** [airbytehq/airbyte#29165] [Connector Questions] [answered]
Hello Airbyte community
I am interested if it's supported by Airbyte to read private Slack channels.

I've followed the quide and it works pretty well for public Slack channels.
For private channels do not get any info even if adding the Bot specifically in it.

Thanks.

**Comment by irvingpop** [comment, marked as answer]
hey @alexopenline this is fixed with v1.1.0 of the slack connector: https://github.com/airbytehq/airbyte/pull/37332
